util/jenkins/ansible-provision.sh

```diff
@@ -218,6 +218,10 @@ if [[ -z authn ]]; then
   authn="false"
 fi
 
+if [[ ! -z authn ]]; then
+  EDXAPP_ENABLE_AUTHN_MICROFRONTEND: true
+fi
+
```

Comment:
No need for if condition, simply do this and place it around other edxapp variables defined in this file:
```suggestion
EDXAPP_ENABLE_AUTHN_MICROFRONTEND: $authn
```